pkg/controller/data_upload_controller.go

```diff
@@ -806,11 +807,20 @@ func (r *DataUploadReconciler) setupExposeParam(du *velerov2alpha1api.DataUpload
 			accessMode = exposer.AccessModeBlock
 		}
 
+		hostingPodLabels := map[string]string{velerov1api.DataUploadLabel: du.Name}
+		for _, k := range util.ThirdPartyLabels {
+			if v, err := nodeagent.GetLabelValue(context.Background(), r.kubeClient, du.Namespace, k); err != nil {
+				r.logger.WithError(err).Warnf("Failed to check existence of label from node-agent, skip adding label %s", k)
+			} else if v != "" {
+				hostingPodLabels[k] = v
```

Comment:
If label whitelist contains `key1` and node-agent is labeled with `key1=""`, the label isn't added into the backup/restore pod

Reply:
Done